meson_options.txt

```diff
@@ -0,0 +1,2 @@
+option('rdrand', type: 'boolean', value: false, description: 'Enable RDRAND Hardware RNG Hash Seed')
+option('shared_lib', type: 'boolean', value: false)
```

Comment:
https://mesonbuild.com/Reference-manual.html#library

Looks like there is a standard option that does that, `default_library`? Do you really need a different one here?

Reply:
I was just following the recommendation of the wrap documentation: https://mesonbuild.com/Wrap-best-practices-and-tips.html#make-libraries-buildable-both-as-static-and-shared
I wouldn't be surprised if that was written before `default_library` was added.

Reply:
Confirmed :) Could you please update to use `default_library`?

```none
<sarum9in> jpakkane, why do we have a recommendation to use option named 'shared_lib'[1] if there is a standard default_library option? [1] https://mesonbuild.com/Wrap-best-practices-and-tips.html#make-libraries-buildable-both-as-static-and-shared
<jpakkane> Probably because that web page was written before the default_library option was added....
```

Reply:
Done